modular_zubbers/code/modules/reagents/reagent_containers/synth_pen.dm

```diff
@@ -0,0 +1,56 @@
+/obj/item/reagent_containers/hypospray/medipen/survival/luxury/synth
+	name = "synth emergency repair medipen"
+	desc = "Cutting edge bluespace technology allowed Nanotrasen to compact 60u of volume into a single medipen. Contains rare and powerful chemicals used to aid synthetic species in exploration of very hard enviroments. WARNING: DO NOT MIX WITH EPINEPHRINE OR ATROPINE."
+	inhand_icon_state = "nanite_hypo"
+	icon_state = "nanite_hypo"
```

Comment:
would this not cause confusion as it may be mistaken for a combat injector vs a medipen?

Reply:
The icon state is not used anywhere, I think, its bright green. Eh, if it will become a problem, I can recolor old one.

Reply:
erol these type of injector look very similar to the ert combat injector

Reply:
> erol these type of injector look very similar to the ert combat injector

its a placeholder for the time being, its a one time use pen which you use and never care about it again.